infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -98,6 +105,12 @@
      */
     private final Map<UUID, Long> streamTailToGlobalTailMap = new HashMap<>();
 
+    /**
+     * Per streams map and their corresponding address space (an address space is defined by the stream's addresses
+     *  and its latest trim mark)
+     */
+    private final Map<UUID, StreamAddressSpace> streamsAddressSpaceMap = new ConcurrentHashMap<>();
```

Comment:
please change Map to ConcurrentMap: private final `Concurrent`Map<UUID, StreamAddressSpace>

Reply:
The sequencer server is single threaded no need to make this map a concurrentMap.

Reply:
so, then `new HashMap<>();`